tasks/volume-present.yaml

```diff
@@ -0,0 +1,69 @@
+- name: Create logical volume
+  lvol:
+    vg: "{{ item[1].vg | default(capra_lvm_filesystem_vg) }}"
+    lv: "{{ item[1].name | mandatory }}"
+    size: "{{ item[1].size | human_to_bytes }}B"
+    opts: "{{ capra_lvm_filesystem_lvcreate_options }}"
+    state: present
+  with_indexed_items: "{{ capra_lvm_filesystem_specs }}"
+
+- name: Create filesystem
+  filesystem:
+    fstype: "{{ item[1].fstype | default(capra_lvm_filesystem_fstype) }}"
+    dev: "/dev/{{ item[1].vg | default(capra_lvm_filesystem_vg) }}/{{ item[1].name }}"
+    opts: "{{ capra_lvm_filesystem_filesystem_options }}"
+  register: r_capra_lvm_filesystem_fs
+  with_indexed_items: "{{ capra_lvm_filesystem_specs }}"
+
+- name: Look up volume UUIDs
+  command: >-
+    /sbin/blkid
+    -c /dev/null
+    -o value
+    -s UUID
+    {{ '/dev/' ~ r_capra_lvm_filesystem_vg ~ '/' ~ item[1].name | quote }}
+  changed_when: false
+  register: r_capra_lvm_filesystem_blkid
+  with_indexed_items: "{{ capra_lvm_filesystem_specs }}"
+  tags: [always]
+  check_mode: false
+  when: (item.state | default('present')) == 'present'
+  vars:
+    r_capra_lvm_filesystem_vg: "{{ item[1].vg | default(capra_lvm_filesystem_vg) }}"
+
+- name: Mount volume
+  mount:
+    path: "{{ item[1].path | default(capra_lvm_filesystem_parent_path ~ '/' ~ item[1].name) }}"
+    src: "UUID={{ item[0].stdout }}"
+    fstype: "{{ item[1].fstype | default(capra_lvm_filesystem_fstype) }}"
+    opts: "{{ item[1].options | default(capra_lvm_filesystem_options) }}"
+    state: mounted
+  register: r_capra_lvm_filesystem_mount
+  with_together:
+    - "{{ r_capra_lvm_filesystem_blkid.results }}"
+    - "{{ capra_lvm_filesystem_specs }}"
```

Comment:
`r_capra_lvm_filesystem_blkid.results` should already contain the specs as part of the input item (use the `debug` module and you should see the structure).